Makefile

```diff
@@ -44,7 +44,7 @@ ifndef MYSQL_PORT_3306_TCP_ADDR
 	MYSQL_PORT_3306_TCP_ADDR = 127.0.0.1
 endif
 
-KIT_VERSION = "\
+KIT_VERSION = "-w -s \
```

Comment:
`KIT_VERSION` is only used to define LDFLAGS that are used to define the version of the binary. Let's not define them in the same variable.

Reply:
@marpaia The reasoning was specific for container sizes -- a Kubernetes manifest that defines multiple images can really add up in terms of data transfer overhead.

Reply:
@tonylambiris yeah, good point. We built the fleet containers via `make docker-build-release`. Maybe we can add these ldflags to that make command to that the binaries used to build the container can be as small as possible.

maybe something like `STRIPPED_SYMBOLS = "-w -s ${KIT_VERSION}"` and use `STRIPPED_SYMBOLS` instead of `KIT_VERSION` in `make binary-bundle` and `make docker-build-release`?